README.md

```diff
@@ -1,10 +1,16 @@
-# hubot-help
+Hubot Help Plugin
+==================
+
+[![Version](https://img.shields.io/npm/v/hubot-help.svg)](https://www.npmjs.com/package/hubot-help)
+[![Downloads](https://img.shields.io/npm/dt/hubot-help.svg)](https://www.npmjs.com/package/hubot-help)
```

Comment:
I would leave the `Downloads` badge out. Instead I would add a badge which shows if dependencies are up-to-date. But in general I’d suggest to keep the badges to a minimum.